test/extended/openstack/dualstack.go

```diff
@@ -85,7 +85,9 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 
 				g.By("Creating Openshift deployment")
 				labels := map[string]string{"app": "lb-dualstack-dep"}
-				testDeployment := createTestDeployment("lb-dualstack-dep", labels, 2, protocolUnderTest, 8081)
+				dep := NewDeploymentOpts()
+				dep = deploymentOpts{Name: "lb-dualstack-dep", Labels: labels, Protocol: protocolUnderTest, Port: 8081}
+				testDeployment := createTestDeployment(dep)
```

Comment:
I still think `NewDeploymentOpts()` is an antipattern.

These three lines could be replaced by a readable and explicit construction of the struct:

```suggestion
				testDeployment := createTestDeployment(deploymentOpts{
					Name:     "lb-dualstack-dep",
					Labels:   labels,
					Replicas: 2,
					Protocol: protocolUnderTest,
					Port:     8081,
				})
```

There is no reason to hide the fact that you want 2 replicas.

Also, building the struct right there even has something of a Pythonic smell if you think about it :P